client/src/plus/updates/index.tsx

```diff
@@ -245,17 +245,14 @@ function GroupComponent({ group }: { group: Group }) {
           })}
         </time>
       </header>
-      {collapseEvents(events.added).map((event) => (
-        <EventComponent key={event.path} event={event} status={"added"} />
-      ))}
-      {collapseEvents(events.removed).map((event) => (
-        <EventComponent key={event.path} event={event} status={"removed"} />
+      {collapseEvents(allEvents).map((event) => (
+        <EventComponent key={event.path} event={event} status={event.status} />
```

Comment:
@Guyzeroth Do you know why we used to show added and removed features separately?